app/util/geo-utils.coffee

```diff
@@ -98,6 +98,25 @@ getDistanceToNearestStop = (lat, lon, stops) ->
     distance: minDist
   }
 
+getDistanceToFurthestStop = (lat, lon, stops) ->
```

Comment:
This could be simplified a bit with writing it something like 

```
getDistanceToFurthestStop = (coordinates, stops) ->
  stops.map stop ->
    stop: stop
    distance: coordinates.distanceTo new L.LatLng stop.lat, stop.lon
  .reduce (previous, current) ->
    if current.distance > previous.distance then current else previous
  , stop: null, distance: 0
```